### PR TITLE
Bump the version number to 0.1.852 for Android

### PR DESCRIPTION
**Pull request by cco3**

### CODE REVIEW

**Comment by g-ortuno** [comment]
Is this in preparation for a release?

File: android/PhysicalWeb/app/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ android {
         minSdkVersion 19
         targetSdkVersion 21
```
Comment (g-ortuno) [inline android/PhysicalWeb/app/build.gradle]: You probably want to bump to 22 as well.

**Comment by cco3** [comment]
Yes, this is in preparation for a release.

**Comment by cco3** [comment]
I accidentally closed this.

**Comment by g-ortuno** [comment]
LGTM

**Comment by cco3** [comment]
Thanks!

**Pull request merged**